pkg/pve-node-driver/l10n/en-us.yaml

```
@@ -50,7 +50,9 @@ cluster:
           label: Processor cores
         memory:
           label: Memory
-        memoryBalloon:
+      memoryBalloon:
+        header: Memory ballooning
+        minimumMemory:
           label: Minimum memory (balloon target)
```

```suggestion
      memoryBalloon:
        header: Memory ballooning
        minimumMemory:
          label: Minimum memory (balloon target)
          tooltip: When set to 0, disables ballooning. When left empty, value from "Memory" is assigned as ballooning target.
```
Maybe we should add a tooltip to this field? 🤔